packages/p2p/src/components/my-profile/payment-methods/payment-method-card/payment-method-card.jsx

```diff
@@ -68,7 +83,16 @@ const PaymentMethodCard = ({
             style={style}
         >
             <div className='payment-method-card__header'>
-                <Icon className='payment-method-card__icon' icon={icon_method} size={medium || small ? 16 : 24} />
+                {icon ? (
+                    <Icon className='payment-method-card__icon' icon={icon} size={medium || small ? 16 : 24} />
+                ) : (
+                    <PaymentMethodIcon
+                        className='payment-method-card__icon'
+                        display_name={payment_method?.display_name}
```

Comment:
would this work as well?

```suggestion
                        display_name={payment_account_name}
```